CodeForAfrica/wazimap-ng
# Update staging by syncing with upstream

## Description
This PR updates our version of the Wazimap-ng backend with the latest changes from the upstream repo. It also maintains the addition of the `display_format` field in the `ProfileKeyMetrics` model that is used internally and resolves all conflicts arising from that change.
Other changes are in regards to migrations, the PR merges some diverging leaves of root migrations following the addition of our custom field as explained above.
_**Some tests are failing**_ and this is because they are failing even on the upstream branch, I've fixed some of them that were failing due to the addition of the `display_format` field and other easy to catch errors. Considering opening a PR to push the changes upstream??
View commits for full changes after conflict resolution, including the fixing of some of the failing test files. 

## Related Issue

https://github.com/CodeForAfrica/wazimap-ng/issues/4

## How to test it locally
Run the docker compose command as shown in the `README.md` and verify it's working as expected.

## Changelog

### Added
- `AWS_QUERYSTRING_AUTH` to readme and updated it's location to the main settings file. Changed read mode from `os.environ.get` to `env.str` to align with upstream.

### Updated
- Location of `DISPLAY_FORMAT_CHOICES` constant to align with upstream changes of moving it to `constants.py` file.
- Resolved conflicts in multiple files resulting from the addition of the `display_format` field.

### Removed
- Additional settings files to align with upstream changes.

## Checklist


### Pull Request


### Commits


### Code Quality


### Testing

- [x]  ✅ ran tests locally & are passing - Mostly
